src/main/kotlin/exercise1/task5/ReverseInteger.kt

```diff
@@ -27,10 +29,10 @@ package org.jetbrains.exercise1.task5
  */
 
 internal fun reverseInteger(x: Int): Int {
-    TODO("Implement me!!!")
+    require(x in -1000000..1000000) { "Number must be integer in range -1000000..1000000" }
+    return if (x < 0) ("-" + abs(x).toString().reversed()).toInt() else x.toString().reversed().toInt()
```

Comment:
In this particular case, 
```abs(x).toString().reversed()).toInt()``` and ```x.toString().reversed().toInt()``` expressions always provide the same result, so it's a safe to store a ```abs(x).toString().reversed()).toInt()``` value to a variable and only prefix the `-` sign in case number is negative.